tests/python/contrib/test_ethosu/test_type_inference.py

```diff
@@ -171,5 +180,26 @@ def test_ethosu_pooling_invalid_dtype():
         run_opt_pass(func, relay.transform.InferType())
 
 
+def test_ethosu_identity_invalid_shape():
```

Comment:
I think we need to test the valid cases as well.

Reply:
Done